turbo/trie/trie_root.go

```diff
@@ -224,15 +224,22 @@ func (l *FlatDBTrieLoader) CalcTrieRoot(tx kv.Tx, quit <-chan struct{}) (libcomm
 		if err != nil {
 			return EmptyRoot, err
 		}
+		var firstPrefix []byte
+		var done bool
 		if accTrie.SkipState {
 			goto SkipAccounts
 		}
 
-		for k, kHex, v, err1 := accs.Seek(accTrie.FirstNotCoveredPrefix()); k != nil; k, kHex, v, err1 = accs.Next() {
```

Comment:
In the event that `FirstNotCoveredPrefix` returns `0x0` because the last trie entry is of the form `0xf..f0..0` then this loop inappropriately re-processes account keys.